routers/admin.py

```diff
@@ -75,6 +75,16 @@ def get_revoked_users(client: Annotated[Auth0Client, Depends(get_auth0_client)],
     resp = client.get_revoked_users(page=pagination.page, per_page=pagination.per_page)
     return resp
 
+@router.get("/users/unverified", response_model=list[Auth0UserData])
+def get_unverified_users(
+    client: Annotated[Auth0Client, Depends(get_auth0_client)],
+    pagination: Annotated[PaginationParams, Depends(get_pagination_params)],
+):
+    """
+    Return users whose email is not verified.
+    """
+    resp = client.get_users(page=pagination.page, per_page=pagination.per_page)
```

Comment:
You should be able to search for unverified users in the Auth0 management API: https://auth0.com/docs/manage-users/user-search/retrieve-users-with-get-users-endpoint (see examples on that docs page). If you can do it in the API call it'll be a lot more efficient and easier to work with than filtering the results after you've got them.